src/Simulation/Core/Submitters/SubmissionOptions.cs

```diff
@@ -60,7 +65,12 @@ public class SubmissionOptions
             int? shots = null,
             ImmutableDictionary<string, string>? inputParams = null,
             string? targetCapability = null) =>
-            new SubmissionOptions(
-                friendlyName ?? FriendlyName, shots ?? Shots, inputParams ?? InputParams, targetCapability ?? TargetCapability);
+            this with
```

Comment:
I'd rather remove `With`, since it 's redundant.

Reply:
Sounds good, my only thought was to not break existing callers, but happy to go on and pull that method, then.

Reply:
To unblock e2e, would it be OK to mark this and `Default` as obsolete for now?